src/time-picker/time-picker.jsx

```diff
@@ -32,6 +33,7 @@ let TimePicker = React.createClass({
     return {
       defaultTime: emptyTime,
       format: 'ampm',
+      pedantic: true,
```

Comment:
Thanks @joewalker - I think we should default this to false since setting it to true is somewhat of a breaking change. Everything else looks good.